src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerImplTest {
+    private static OperationHandler operationHandler;
+    private Transaction transaction;
+    private Fruit fruit;
+
+    @BeforeClass
+    public static void beforeAll() {
+        operationHandler = new BalanceOperationHandlerImpl();
+    }
+
+    @Before
+    public void setUp() {
+        fruit = new Fruit("mango");
+        transaction = new Transaction(Transaction.Operation.BALANCE, fruit, 52);
+    }
+
+    @Test
+    public void operation_ApplyBalance_ok() {
```

Comment:
```suggestion
    public void operation_applyBalance_ok() {
```
please check all test method names in all test classes
they all should have the same naming pattern